new-lamassu-admin/src/routing/routes.js

```diff
@@ -10,6 +10,8 @@ import {
 
 import { AppContext } from 'src/App'
 import AuthRegister from 'src/pages/AuthRegister'
+// import should appear before cashout (why?)
```

Comment:
You can remove this comment. I think I already explained this in a call, but imports are organized by scope and alphabetically.